cyberglove/src/serial_glove.cpp

```diff
@@ -148,10 +151,15 @@ namespace cyberglove
   void CybergloveSerial::stream_callback(char* world, int length)
   {
     //read each received char.
+    const char r_state[2] = {'I', 'R'};
```

Comment:
This would crash with 16bit protocol, as there are more than 2 states there.
I suggest that you simply print the state number, instead of a letter.

Reply:
good point